app/com/arpnetworking/kairos/service/KairosDbServiceImpl.java

```diff
@@ -234,56 +237,64 @@ private static MetricsQuery useAvailableRollups(
                         return Metric.Builder.<Metric, Metric.Builder>clone(metric)
                                 .setName(metricName.substring(0, metricName.length() - 2))
                                 .build();
+                    }
```

Comment:
The following section, up until the `maxUnit.isPresent()` code, is me repeatedly applying Intellij's "remove redundant else" refactor